test/e2e/config/docker.yaml

```
@@ -354,9 +354,9 @@ variables:
   # The following Kubernetes versions should be the latest versions with already published kindest/node images.
   # This avoids building node images in the default case which improves the test duration significantly.
   KUBERNETES_VERSION_MANAGEMENT: "v1.30.0"
-  KUBERNETES_VERSION: "v1.30.0"
-  KUBERNETES_VERSION_UPGRADE_FROM: "v1.29.4"
-  KUBERNETES_VERSION_UPGRADE_TO: "v1.30.0"
+  KUBERNETES_VERSION: "v1.31.0-beta.0"
+  KUBERNETES_VERSION_UPGRADE_FROM: "v1.30.2"
+  KUBERNETES_VERSION_UPGRADE_TO: "v1.31.0-beta.0"
   KUBERNETES_VERSION_LATEST_CI: "ci/latest-1.31"
   ETCD_VERSION_UPGRADE_TO: "3.5.12-0"
```

```suggestion
  ETCD_VERSION_UPGRADE_TO: "3.5.14-0"
```

https://github.com/kubernetes/kubernetes/blob/v1.31.0-beta.0/cmd/kubeadm/app/constants/constants.go#L325

If I see correctly